pyopencl/array.py

```diff
@@ -2579,6 +2579,38 @@ def hstack(arrays, queue=None):
 
     return result
 
+
+def stack(arrays, axis=0, queue=None):
```

Comment:
Add to docs.

Reply:
f7c6bff226a55c1e4b95cf71bd819ffb9fe1c8f9